examples/test_enterprise_management_v1_examples.py

```diff
@@ -0,0 +1,443 @@
+        try:
+            parent = 'crn:v1:bluemix:public:enterprise::a/' + enterprise_account_id + '::account-group:' + first_account_group_id
+
+            # begin-create_account
+
+            create_account_response = enterprise_management_service.create_account(
+                parent=parent,
+                name=account_name,
+                owner_iam_id=enterprise_account_iam_id,
+            ).get_result()
+
+            print(json.dumps(create_account_response, indent=2))
+
+            # end-create_account
+
+            global account_id
+            account_id = create_account_response.get('account_id')
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @pytest.mark.skip(reason='Skip by design.')
+    @needscredentials
+    def test_import_account_to_enterprise_example(self):
+        """
+        import_account_to_enterprise request example
+        """
+        print('\nimport_account_to_enterprise() result:')
+
+        try:
+            # begin-import_account_to_enterprise
+
+            response = enterprise_management_service.import_account_to_enterprise(
+                enterprise_id=enterprise_id,
+                account_id='standalone_account_id',
+            ).get_result()
+
+            print(json.dumps(response, indent=2))
+
+            # end-import_account_to_enterprise
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_list_accounts_example(self):
+        """
+        list_accounts request example
+        """
+        assert enterprise_id is not None
+
+        print('\nlist_accounts() result:')
+
+        try:
+            # begin-list_accounts
+
+            list_accounts_response = enterprise_management_service.list_accounts(
+                enterprise_id=enterprise_id,
+            ).get_result()
+
+            print(json.dumps(list_accounts_response, indent=2))
+
+            # end-list_accounts
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_get_account_example(self):
+        """
+        get_account request example
+        """
+        assert account_id is not None
+
+        print('\nget_account() result:')
+
+        try:
+            # begin-get_account
+
+            account = enterprise_management_service.get_account(
+                account_id=account_id,
+            ).get_result()
+
+            print(json.dumps(account, indent=2))
+
+            # end-get_account
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_update_account_example(self):
+        """
+        update_account request example
+        """
+        assert account_id is not None
+        assert second_account_group_id is not None
+
+        print('\nupdate_account() result:')
+
+        try:
+            new_parent = 'crn:v1:bluemix:public:enterprise::a/' + enterprise_account_id + '::account-group:' + second_account_group_id
```

Comment:
```suggestion
            new_parent = 'crn:v1:bluemix:public:enterprise::a/' + enterprise_account_id + '::account-group:' + new_parent_account_group_id
```
I think you could also move this statement below the begin comment.  I think the initialization of "new_parent" would be useful to help the user understand the code example better.